src/main/java/org/ccci/util/mail/MailMessage.java

```diff
@@ -38,22 +40,31 @@
 public class MailMessage 
 {
     private Session session;
-    private Message message;
+    private MimeMessage message;

```

Comment:
Does this have to change?
```suggestion
    private Message message;
```

Reply:
Yes, Message does not support non-ascii characters

Reply:
Ah, I see. It in fact **is** a `MimeMessage` here https://github.com/CruGlobal/util/blob/issue-822110-email-chars/src/main/java/org/ccci/util/mail/MailMessage.java#L59, and could be declared a simple `Message` here https://github.com/CruGlobal/util/blob/issue-822110-email-chars/src/main/java/org/ccci/util/mail/MailMessage.java#L43, but would not be able to have the charset set here https://github.com/CruGlobal/util/blob/issue-822110-email-chars/src/main/java/org/ccci/util/mail/MailMessage.java#L273 if declared `Message` (even though it actually **is** a `MailMessage`) because the compiler.

Reply:
You could typecast, but not good practice (and not necessary):
```
((MimeMessage)message).setText(message.getContent().toString(), charset.toString());
```